nmap.go

```diff
@@ -1149,6 +1149,29 @@ func WithBadSum() func(*Scanner) {
 
 /*** Output ***/
 
+// WithVerbosity sets and increases the verbosity level of nmap.
+func WithVerbosity(level int) func(*Scanner) {
+
+	return func(s *Scanner) {
+		if level < 0 || level > 10 {
+			panic("value given to nmap.WithVerbosity() should be between 0 and 10")
+		}
+		s.args = append(s.args, "-v")
```

Comment:
WDYT of doing it in one line?

```suggestion
		s.args = append(s.args, fmt.Sprintf("-v%d", level))
```

Reply:
I like this implementation better.